src/noise.h

```diff
@@ -94,6 +99,9 @@ class PcgRandom {
 	void bytes(void *out, size_t len);
 	s32 randNormalDist(s32 min, s32 max, int num_trials=6);
 
+	// Allow save and restore of state
+	void getState(u64 &state, u64 &inc);
```

Comment:
also how about passing a `u64[2]` to make it more opaque? 
```suggestion
	void getState(u64 &state, u64 &inc); const
```

Reply:
applied